src/app/stores/migrator.ts

```diff
@@ -636,4 +645,154 @@ export class Migrator {
+              element.properties.yData.orderByExpression =
+                element.properties.yData.expression;
+            }
+          }
+        }
+
+        if (
+          Prototypes.isType(item.chartElement.classID, "plot-segment.polar")
+        ) {
+          const element = item.chartElement as PlotSegment<PolarProperties>;
+          if (element.properties.xData) {
+            element.properties.xData = this.updateAxis(
+              element.properties.xData
+            );
+          }
+          if (element.properties.xData === undefined) {
+            element.properties.xData = null;
+          } else {
+            element.properties.xData.orderByCategories =
+              element.properties.xData.categories;
+            element.properties.xData.orderByExpression =
+              element.properties.xData.expression;
+          }
+          if (element.properties.yData) {
+            element.properties.yData = this.updateAxis(
+              element.properties.yData
+            );
+          }
+          if (element.properties.yData === undefined) {
+            element.properties.yData = null;
+          } else {
+            element.properties.yData.orderByCategories =
+              element.properties.yData.categories;
+            element.properties.yData.orderByExpression =
+              element.properties.yData.expression;
+          }
+        }
+        if (Prototypes.isType(item.chartElement.classID, "plot-segment.line")) {
+          const element = item.chartElement as PlotSegment<LineGuideProperties>;
+          if (element.properties.axis) {
+            element.properties.axis = this.updateAxis(element.properties.axis);
+            element.properties.axis.orderByExpression =
+              element.properties.axis.expression;
+            element.properties.axis.orderByCategories =
+              element.properties.axis.categories;
+          }
+        }
+        if (
+          Prototypes.isType(item.chartElement.classID, "plot-segment.curve")
+        ) {
+          const element = item.chartElement as PlotSegment<CurveProperties>;
+          if (element.properties.xData) {
+            element.properties.xData = this.updateAxis(
+              element.properties.xData
+            );
+          }
+          if (element.properties.xData === undefined) {
+            element.properties.xData = null;
+          } else {
+            element.properties.xData.orderByCategories =
+              element.properties.xData.categories;
+            element.properties.xData.orderByExpression =
+              element.properties.xData.expression;
+          }
+          if (element.properties.yData) {
+            element.properties.yData = this.updateAxis(
+              element.properties.yData
+            );
+          }
+          if (element.properties.yData === undefined) {
+            element.properties.yData = null;
+          } else {
+            element.properties.xData.orderByCategories =
```

Comment:
Looks like this could be a type-o. Should this be `yData`?

Reply:
Fixed, thanks!